source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py

```diff
@@ -611,8 +611,8 @@ def _load_extensions(self):
             if self._livestream == 1:
                 # Enable Native Livestream extension
                 # Default App: Streaming Client from the Omniverse Launcher
-                enable_extension("omni.kit.streamsdk.plugins-4.5.1")
-                enable_extension("omni.kit.livestream.core-4.3.6")
+                enable_extension("omni.kit.streamsdk.plugins-3.2.1")
+                enable_extension("omni.kit.livestream.core-3.2.0")
                 enable_extension("omni.kit.livestream.native-4.1.0")
```

Comment:
Do we need to even specify the version here? Can't we just enable the extension and it gets whatever version is present? I am not sure if the extension's name itself has version in it.

Reply:
I have tested it without the vesion call outs and it doesn't work for `omni.kit.streamsdk.plugins` and `omni.kit.livestream.core`, but if I remove version from `omni.kit.livestream.native` it does work.

Reply:
I have experienced the same performance as @jtigue-bdai. I think that we should just adhere to whatever versions are specified in `/isaac-sim/apps/omni.isaac.sim.headless.native.kit`, as NV probably knows best which packages versions are compatible

Reply:
Wondering if we should read the extension versions from that file so that we don't need to modify the settings here for different Isaac Sim versions. But this can be its own MR. If the current fix works for 4.1, we can move ahead with it.